app/uk/gov/hmrc/vatapi/models/Errors.scala

```diff
@@ -98,5 +115,14 @@ object Errors {
     }
   }
 
+  def businessJsonError(error: Errors.Error) = Json.toJson(Errors.businessError(error))
```

Comment:
As these 3 methods aren't being used they can be removed and we'll pick this kind of refactoring up when we implement the new code design